account_cutoff_base/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,17 @@
+- Alexis de Lattre \<<alexis.delattre@akretion.com>\>
+
+- Alexandre Fayolle \<<alexandre.fayolle@camptocamp.com>\>
+
+- Stéphane Bidoul (ACSONE)
+
+- Adrien Peiffer (ACSONE)
+
+- Pedro M. Baeza \<<pedro.baeza@gmail.com>\>
+
+- Jeroen Evens \<<jeroen.evenss@dynapps.be>\>
+
+- Jim Hoefnagels \<<jim.hoefnagels@dynapps.be>\>
+
+- [Trobz](https://trobz.com):  
+  - Dzung Tran \<<dungtd@trobz.com>\>
+  - Thao Le <thaolt@trobz.com>
```

Comment:
```suggestion
  - Thao Le \<<thaolt@trobz.com>\>
```
Then regenerating the readme file

Reply:
@dzungtran89  the PR has been updated! Thank you.